scripts/check-websockets.ts

```diff
@@ -41,6 +42,15 @@ function checkIsWebSocketAlive({
       ws.close();
       resolve(true);
     });
+
+    ws.on('close', (code: number, reason: string) => {
+      if (code !== 1000) {
+        console.log(
+          `${chainKey} WebSocket connection to ${endpoint} closed with code ${code} and reason: ${reason}`,
+        );
+        resolve(false);
```

Comment:
not sure maybe better to do reject(`reason`) here.

Reply:
I wanted to differentiate it from the actual errors of the endpoint. In these cases the websocket doesn't throw an error, but doesn't open the connection either, so it is a different 'category'